Mathlib/RingTheory/Finiteness/Defs.lean

```diff
@@ -134,6 +134,14 @@ end Finite
 
 end Module
 
+instance AddMonoid.FG.to_moduleFinite_nat {M : Type*} [AddCommMonoid M] [FG M] :
+    Module.Finite ℕ M :=
+  Module.Finite.iff_addMonoid_fg.mpr ‹_›
+
+instance AddMonoid.FG.to_moduleFinite_int {G : Type*} [AddCommGroup G] [FG G] :
```

Comment:
Why use `AddMonoid.FG` instead of `AddGroup.FG` here? Are there instances in any direction?

Reply:
cf [Zulip](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/docs.23Group.2EFG.20vs.20docs.23Monoid.2EFG). TLDR we will probably drop `AddGroup.FG` in favor of `AddMonoid.FG` in the future (but keeping `AddGroup.FG` as the name). 

> Are there instances in any direction

No, which is why the current situation is so damageable

Reply:
Ok, thanks for the reference to the Zulip thread. Could you add it to the PR description?